flight/Libraries/sanitycheck.c

```diff
@@ -101,11 +101,6 @@ int32_t configuration_check()
 
 	for(uint32_t i = 0; i < num_modes; i++) {
 		switch(modes[i]) {
-			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_MANUAL:
```

Comment:
OK, will add manual mode back. For future reference, we should have a warning for cpheli vehicles using manual mode.